CommunityCenterDB/5 reports.sql

```diff
@@ -0,0 +1,31 @@
+--1)	For corporate events, list all venders in the following format: vender, satisfaction score – satisfaction description, cause (sub-cause), $payment amount 
+    --(Target, 1 - gold, cause (sub-cause), $100.00). Show them with the most satisfactory on top.
+select Vendors = concat('(', e.VendorName, ', ', e.SatisfactionRate, ' - ', e.SatisfactionDesc, ', ', e.Cause, ' (', e.SubCause, '), $', e.PaymentAmount, ')')
```

Comment:
Although the prentices around the whole concat had not been requested, this format is also okay.

Reply:
Your example provided has parentheses so I assumed you wanted them: (Target, 1 - gold, cause (sub-cause), $100.00).